mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -23,42 +25,61 @@ class ExperimentPage extends Component {
     dispatchSearchRuns: PropTypes.func.isRequired,
   };
 
-  state = {
-    paramKeyFilter: new KeyFilter(),
-    metricKeyFilter: new KeyFilter(),
+  static defaultState = {
+    paramKeyFilterString: "",
+    metricKeyFilterString: "",
     getExperimentRequestId: getUUID(),
     searchRunsRequestId: getUUID(),
     searchInput: '',
     lastExperimentId: undefined,
     lifecycleFilter: LIFECYCLE_FILTER.ACTIVE,
   };
 
+  static getStore(experimentId) {
+    return LocalStorageUtils.getStore("ExperimentPage", experimentId);
+  }
+
+  store = ExperimentPage.getStore(this.props.experimentId);
+  state = LocalStorageUtils.loadComponentState(this.store, ExperimentPage.defaultState);
```

Comment:
I think it would be more reasonable if the expectation is that the store will store the same keys in the dictionary as `defaultState` (i.e., that it looks more like a class than one dictionary which is the subset of another).